Makefile

```diff
@@ -8,23 +8,29 @@ LUAJIT_DIR=/usr/local/openresty/luajit
 
 CC=gcc
 CFLAGS=-g -fPIC -O2
-XMLSEC1_CFLAGS=$(shell xmlsec1-config --cflags --crypto=openssl)
+XMLSEC1_CFLAGS=-D__XMLSEC_FUNCTION__=__func__ -DXMLSEC_NO_SIZE_T -DXMLSEC_NO_GOST=1 -DXMLSEC_NO_GOST2012=1 -DXMLSEC_NO_CRYPTO_DYNAMIC_LOADING=1 -Ixmlsec1-1.2.28/include/ -I/usr/include/libxml2 -DXMLSEC_CRYPTO_OPENSSL=1
 CFLAGS_ALL=$(CFLAGS) -Wall -Werror -std=c99 $(XMLSEC1_CFLAGS)
 LIBFLAG=-shared
 LDFLAGS=-g -O2
-XMLSEC1_LDFLAGS=$(shell xmlsec1-config --libs --crypto=openssl)
+XMLSEC1_STATIC_LIBS=xmlsec1-1.2.28/./src/openssl/.libs/libxmlsec1-openssl.a xmlsec1-1.2.28/./src/.libs/libxmlsec1.a
+XMLSEC1_LDFLAGS=-lxml2 -lssl -lcrypto -ldl -Wl,--whole-archive $(XMLSEC1_STATIC_LIBS) -Wl,--no-whole-archive -lxslt
```

Comment:
Just to make sure, we still need to add libxml to build APISIX, right?

Reply:
yes.
For ubuntu 20.04: libxml2-dev libxslt-dev libssl-dev
For centos7: libxml2-devel openssl-devel libxslt-devel